sign-up.html

```diff
@@ -1,28 +1,49 @@
 <!DOCTYPE html>
 <html lang="he">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Sign Up</title>
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+        crossorigin="anonymous">
     <link rel="stylesheet" href="styles/style.css">
 </head>
+
 <body>
-    <nav class="navbar navbar-static-top">
-        <a class="navbar-brand" href="index.html">OfekTwitter</a>
-        <ul class="nav navbar-nav navbar-right">
-            <li><a href="users.html">Users</a></li>
-            <li><a href="sign-up.html">Sign Up</a></li>
-            <li><a href="sign-in.html">Sign In</a></li>
-        </ul>
+    <nav class="navbar">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <a class="navbar-brand" href="index.html">OfekTwitter</a>
+            </div>
+
+            <ul class="nav navbar-nav navbar-right">
+                <li>
+                    <a href="users.html">Users</a>
+                </li>
+                <li>
+                    <a href="sign-up.html">Sign Up</a>
+                </li>
+                <li>
+                    <a href="sign-in.html">Sign In</a>
+                </li>
+            </ul>
+        </div>
     </nav>
 
-    <form class="form">
-        <label>Username <input type="text" name="username" id="username"></label>
-        <label>Password <input type="password" name="password" id="password"></label>
-        <label>Confirm Password <input type="password" name="confirmPassword" id="confirmPassword"></label>
-        <input class="button" type="submit" value="Submit">
-    </form>
+    <div class="container">
+        <form class="form-signin col-md-4 col-md-offset-4 col-sm-4 col-sm-offset-4">
+            <h2 class="form-signin-heading">Sign Up</h2>
+            <label for="inputUsername" class="sr-only">Username</label>
+            <input type="username" id="inputUsername" class="form-control" placeholder="Username" required autofocus>
```

Comment:
you should use the class "input-group" for grouping label + input
and the class "input-group-addon" on the label/span near the input